components/footer.js

```diff
@@ -72,32 +73,32 @@ const Footer = () => (
             <Box p={[1, 0]} mb={[0, 3]} width={[1/2, 1]}>
               <StyledIcon height='32px' />
             </Box>
-            <Box p={[1, 0]} pr={[0, '50%']} width={[1/2, 1]}> <FooterText>Detect and measure internet censorship all around the world.</FooterText> </Box>
+            <Box p={[1, 0]} pr={[0, '50%']} width={[1/2, 1]}> <FooterText>{intl.formatMessage({ id: 'Footer.Text.Slogan' })}</FooterText> </Box>
           </Flex>
         </FooterBox>
         <FooterBox width={[1/2, 1/6]}>
-          <FooterHead>OONI</FooterHead>
-          <FooterLink href='https://ooni.io/about/' label='About' />
-          <FooterLink href='https://ooni.io/post/' label='Blog' />
-          <FooterLink href='#' label='Reports' />
-          <FooterLink href='https://ooni.io/about/#contact' label='Contact Us' />
+          <FooterHead>{intl.formatMessage({ id: 'Footer.Headings.OONI' })}</FooterHead>
+          <FooterLink href='https://ooni.io/about/' label={intl.formatMessage({ id: 'Footer.Links.About' })} />
+          <FooterLink href='https://ooni.io/post/' label={intl.formatMessage({ id: 'Footer.Links.Blog' })} />
+          <FooterLink href='#' label={intl.formatMessage({ id: 'Footer.Links.Reports' })} />
+          <FooterLink href='https://ooni.io/about/#contact' label={intl.formatMessage({ id: 'Footer.Links.Contact' })} />
         </FooterBox>
         <FooterBox width={[1/2, 1/6]}>
-          <FooterHead>Data</FooterHead>
-          <FooterLink href='https://ooni.io/install/' label='Install Probe' />
-          <FooterLink href='https://explorer.ooni.io/' label='Explorer' />
-          <FooterLink href='https://api.ooni.io/' label='API' />
-          <FooterLink href='https://ooni.io/nettest/' label='Test Docs' />
+          <FooterHead>{intl.formatMessage({ id: 'Footer.Headings.Data' })}</FooterHead>
+          <FooterLink href='https://ooni.io/install/' label={intl.formatMessage({ id: 'Footer.Links.Probe' })} />
+          <FooterLink href='https://explorer.ooni.io/' label={intl.formatMessage({ id: 'Footer.Links.Explorer' })} />
+          <FooterLink href='https://api.ooni.io/' label={intl.formatMessage({ id: 'Footer.Links.API' })} />
+          <FooterLink href='https://ooni.io/nettest/' label={intl.formatMessage({ id: 'Footer.Links.Tests' })} />
```

Comment:
<Bikeshedding>
Maybe the key name should be singular similar to `Text` in `Footer.Text.Slogan`, as in `Footer.Link.Tests` (instead of `Footer.Links.Tests`).
Same for `Headings`.
</Bikesheedding>

Reply:
I see there are several instances of this. I would suggest a mass search and replace for `.Headings.` with `.Heading.` and `.Links.` with `.Link.` in the code and in the csv.

Reply:
Updated these.